pigeon/client.py

```diff
@@ -140,8 +162,15 @@ def send(self, topic: str, **data):
 
         """
         self._ensure_topic_exists(topic)
-        serialized_data = self._topics[topic](**data).serialize()
+
+        # Users shouldn't have to worry about what core messages are, so they exist as a special case.
```

Comment:
As soon as you register the topics, the end up in `self._topics`, so I don't see a reason to treat them differently here.